packages/components/src/DateTimePickers/IconButton/IconButton.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import IconButton from './IconButton.component';
+
+describe('IconButton', () => {
+	it('should render', () => {
```

Comment:
Still KO. From my previous review : `Please add some more props for icon and button`